### DISCUSSION TITLE
exchange1c.php

### DISCUSSION THREAD

**Discussion by AlexusTdV** [Tseka85/opencart3-exchange1c#63] [Q&A]
Данный файл открывается с ошибкой 
"Notice: Only variables should be passed by reference in /home/echo-nvrsk/new.echo-nvrsk.ru/docs/export/exchange1c.php on line 13"

На 13 строке 
"eval(reset(preg_grep('/VERSION/', file('../index.php'))));"

Подключения из 1С нет.

**Comment by Tseka85** [comment]
закомментируйте 13 строку

**Comment by Tseka85** [comment]
@ATropichev  ваш коммит ? 

https://github.com/Tseka85/opencart3-exchange1c/pull/48/commits/c63b55baa74bf7440e52411bf1f4d45d3ee3a468

**Reply by ATropichev** [reply]
Сейчас проверим

**Reply by ATropichev** [reply]
Проверил. В [pull request](https://github.com/Tseka85/opencart3-exchange1c/pull/52) эта ошибка [исправлена:](https://github.com/Tseka85/opencart3-exchange1c/pull/52/commits/fdf41a3f65d40657ce806cf42d42ba456fcdc715#diff-c90827a4b13c9e08adbe4b691b9650360aa7eddea8a76dfe33ad563d91e18d33)
